packages/models-library/src/models_library/api_schemas_webserver/clusters.py

```diff
@@ -8,7 +8,7 @@
 class ClusterPathParams(BaseModel):
     cluster_id: ClusterID
     model_config = ConfigDict(
-        populate_by_name=True, 
+        populate_by_name=True,
```

Comment:
not sure I see the point of this change? there is no alias so what does it do?

Reply:
I didn't change anything here. It was like so in `master` as well.